components/StyledLinkSvgContainer/index.js

```diff
@@ -1,6 +1,6 @@
 import Link from "next/link";
 import styled from "styled-components";
-import { infoSVG, callSVG } from "../assets/contactsSVG";
+import { infoSVG } from "../assets/contactsSVG";
 
 const StyledCircularContainer = styled.div`
```

Comment:
```suggestion
const StyledLink = styled(Link)`
  height: 20px;
  width: 20px;
  border-radius: 50%;
  border: none;
  padding: 0;
  margin: 0.3rem;
  background-color: transparent;
`;
```

Instead of working with an extra `div` I think it should also work with styling the `Link` directly. :)

Reply:
changed